common/db.go

```diff
@@ -615,6 +615,20 @@ func (db *DB) UseUnbondingLock(id *big.Int, delegator ethcommon.Address, usedBlo
 	return nil
 }
 
+func (db *DB) LatestUnbondingLockID() (*big.Int, error) {
+	glog.V(DEBUG).Infof("db: Getting latest unbonding lock ID")
+
+	var latestUnbondingLockID int64
+	row := db.dbh.QueryRow("SELECT MAX(id) FROM unbondingLocks")
```

Comment:
Perhaps a more robust way to do this would be to `SELECT COUNT(id) FROM unbondingLocks` and check if that matches `d.nextUnbondingLockId - 1` . That should allow us to catch gaps, eg

Node stops
Event E is missed
Node starts up
Event E+1 is delivered

Then if `COUNT(id) != d.NextUnbondingLockId - 1` , we can iterate over the local ids in order, and query for any missing ids. (This could also probably be done within SQL by joining the unbonding lock table onto itself.)

Reply:
The presented scenario *should* be covered by the processing of historical logs for the `Unbond`, `Rebond` and `WithdrawStake` events by `UnbondingService` when the node starts up - this is based on direct `eth_getLogs` RPC calls which have the same risk of failure when using an Infura WS connection as the RPC calls involved with querying for missing local unbonding lock IDs (in which case, the node would have to retry). 

I do see the usefulness of this solution for the following related scenario:

1. Node starts up and starts subscription for new `Unbond` events
2. `Unbond` subscription breaks
3. `Unbond` event E is emitted and missed by the node
4. Node successfully resubscribes to new `Unbond` events
5. Node receives `Unbond` event E + 1
6. CLI queries for local unbonding locks which do not include `Unbond` event E

The alternative would be to track more state in `UnbondingService` and fetch historical logs before each resubscription, but the proposed solution feels cleaner. I can implement this and hopefully we can remove this additional code once we have more resilient event watching that is synchronized with the last seen block.